src/inference/include/openvino/runtime/core.hpp

```diff
@@ -65,6 +65,7 @@ class OPENVINO_RUNTIME_API Core {
      * @param device_name Device name to identify a plugin.
      * @return A vector of versions.
      */
+    Version get_version() const;
```

Comment:
why do we need it? In order to get OV Runtime version, we don't have to create a Core object and we can use:
https://github.com/openvinotoolkit/openvino/blob/bcc21330497c13672c13a10c1f95075d9bb1a789/src/core/include/openvino/core/version.hpp#L52-L56

Reply:
I've added this so that it is possible to obtain the full Version object in Python API. Without it, to my knowledge, the only getter available returned buildNumber alone, which was not sufficient for me to align the output between the two applications

Reply:
After further analysis, pyopenvino.cpp contains a get_version method that returns the aforementioned buildNumber instead of the full version. I could add additional getter function, such as 'get_full_version', or change the 'get_version' method to return the full object instead, and then remove the new function. I am not certain which option is adequate, though, so any input would be appreciated!

Reply:
Oki, I will roll them back, although I am still missing the function in pyopenvino API to get the full version object/string with full OV information. If changing the Core is not an option, what would be the best way to solve this?